plugins/terraform/layer0/data_source_layer0_api.go

```diff
@@ -32,19 +36,19 @@ func dataSourceLayer0API() *schema.Resource {
 }
 
 func dataSourceLayer0APIRead(d *schema.ResourceData, meta interface{}) error {
-	client := meta.(*Layer0Client)
+	apiClient := meta.(client.Client)
```

Comment:
[nit] the naming of `apiClient` caught me for a second here --- what api? are there other clients? Knowing the context of the function name makes it more clear, but it's still feels like an assumption that it's related to layer0.